test/integration/feed/index.spec.ts

```diff
@@ -15,9 +15,9 @@ function makeChunk(index: number) {
 
 async function uploadChunk(url: string, index: number): Promise<ChunkReference> {
   const chunk = makeChunk(index)
-  const referenceResponse = await chunkAPI.upload(url, chunk.data, getPostageBatch())
+  const reference = await chunkAPI.upload(url, chunk.data, getPostageBatch())
 
-  return hexToBytes(referenceResponse.reference as HexString) as ChunkReference
+  return hexToBytes(reference as HexString) as ChunkReference
```

Comment:
the type assertion is not not necessary here.
```suggestion
  return hexToBytes(reference) as ChunkReference
```